### PR TITLE
Revert "Cherry-pick #2639 Add dynamic layers to tigera infra layer in SG EV-3506"

### PR DESCRIPTION
**Pull request by vara2504**
Reverts tigera/operator#2649

### CODE REVIEW

**Review by rene-dekker — APPROVED** [review]
lgtm

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
